src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -241,6 +241,18 @@ public string OriginalName
             get { return nodeLogic.GetOriginalName(); }
         }
 
+        /// <summary>
+        /// Returns the Package name of the node, if it comes from a package
+        /// </summary>
+        [JsonProperty(Order = 9)]
+        public string PackageName
```

Comment:
why is this being serialized? It looks like you never read the serialized data?

Reply:
You are right, I will just ignore it, apologies!

Reply:
Big thanks for this spot check, @mjkkirschner , hopefully it's fixed with this [PR](https://github.com/DynamoDS/Dynamo/pull/14216)